libmproxy/script/script.py

```diff
@@ -88,6 +88,8 @@ def run(self, name, *args, **kwargs):
             Raises:
                 ScriptException if there was an exception.
         """
+        if not self.ns:
```

Comment:
It is already crashing here in some testcases. This was the simplest fix for me. Maybe you can investigate where this comes from?

```
Exception in thread ProxyThread (0.0.0.0:49486):
Traceback (most recent call last):
  File ".../2.7/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File ".../mitmproxy/test/tservers.py", line 76, in run
    self.tmaster.run()
  File ".../mitmproxy/libmproxy/controller.py", line 123, in run
    self.tick(self.masterq, 0.1)
  File ".../mitmproxy/libmproxy/flow.py", line 809, in tick
    return super(FlowMaster, self).tick(q, timeout)
  File ".../mitmproxy/libmproxy/controller.py", line 108, in tick
    self.handle(*msg)
  File ".../mitmproxy/libmproxy/controller.py", line 130, in handle
    m(obj)
  File ".../mitmproxy/libmproxy/flow.py", line 958, in handle_serverdisconnect
    self.run_script_hook("serverdisconnect", server_conn)
  File ".../mitmproxy/libmproxy/flow.py", line 691, in run_script_hook
    self._run_single_script_hook(script_obj, name, *args, **kwargs)
  File ".../mitmproxy/libmproxy/flow.py", line 685, in _run_single_script_hook
    script_obj.run(name, *args, **kwargs)
  File ".../mitmproxy/libmproxy/script/script.py", line 90, in run
    f = self.ns.get(name)
AttributeError: 'NoneType' object has no attribute 'get'
```